### ISSUE TITLE
The scraperesults of json data is wrongly formated

### ISSUE THREAD

**Issue by wongjiahau** [fesvictor/TCLCPhase1#32]

**Comment by newlyzher** [comment]
can u please describe the format error?

**Comment by wongjiahau** [comment]
A correct `json` format should have a curly brackets that wrap up the entire file

**Comment by newlyzher** [comment]
I don't think so. From what I know, json format is just sorted the attributes and values curly brackets. Doesn't really need curly brackets to wrap it up. However, you may ask Dr. Victor regarding this matter.
